### DISCUSSION TITLE
Installing new Packages, Win, MiKTeX

### DISCUSSION THREAD

**Discussion by LawSteveline** [xournalpp/xournalpp#5672] [Q&A]
Hello, I am a newbi in LaTeX and Xournal++. I used it for one jear without more knowleg. I am using a win 11 dektop and MikTex for LaTeX. I installed MikTex without more packages, but now I need some more. I am trying to make a nice table and ask chatGPT for help.
It gave me this code:
\documentclass{article}
\usepackage{booktabs}
\begin{document}
\begin{table}[htbp]
  \centering
  \caption{Beispiel einer schön formatierten Tabelle mit dem \texttt{booktabs}-Paket}
  \label{tab:schoene-tabelle}
  \begin{tabular}{ccc}
    \toprule
    \textbf{Spalte 1} & \textbf{Spalte 2} & \textbf{Spalte 3} \\
    \midrule
    Wert 1 & Wert 2 & Wert 3 \\
    Wert 4 & Wert 5 & Wert 6 \\
    \bottomrule
  \end{tabular}
\end{table}
\end{document}

but it won't work for me. So I installed at "MiKTeX Console" under "Packages" the "booktabs" packaget.
Do I need to install some more Packages or do I have to do more than just klicking install in MiKTeX?
Do I need to tell Xournal++ in any way that I installed new Packages?

Hope someone can help me a bit. I searced like 6h to solve my problem.

Thanks for answering

Steve

<img width="260" alt="Bild_In_Latex" src="https://github.com/xournalpp/xournalpp/assets/169433128/19b22903-be3b-44a5-a4d1-677ee83c1a92">

<img width="413" alt="Bild_MikTeX" src="https://github.com/xournalpp/xournalpp/assets/169433128/aba961d7-dbe9-467e-9aba-2bebcb96a43b">

**Comment by fengzyf** [comment]
see https://xournalpp.github.io/guide/tools/latex/

> The LaTeX formulas are rendered by substituting the user-provided formula (along with other information like tool color) into a template file. The template file to use can be changed in the LaTeX tab of the Preferences window.
> 
> A default LaTeX template is provided under the name default_template.tex. Clicking on it, you will find its location in your file system (within the [shared resources folder](https://xournalpp.github.io/guide/file-locations/)).
> 
> To generate a custom LaTeX template, make a copy of the default template, save it under a different name, and customize it.
